tests/testthat/test-formatting.R

```diff
@@ -1,6 +1,6 @@
+library(dplyr)
```

Comment:
If this is required it should occur once in the setup-file unless there is a reason to localize it (which in this and virtually all cases, there  isn't)